### PR TITLE
Admin Panel for running benchmarks

### PR DESCRIPTION
**Pull request by shahsaurabh0605**
Added a sample for admin panel for running benchmarks. Currently I have implemented only for the version of releases and not for commits. I am facing the following problem :

Whenever I select the benchmark type and release type the url gets created but it doesn’t showcase any output. How on refreshing the page we will get the required output. How do I solve this?

I have made temporary arrangements to print the output of docker method for now.

### CODE REVIEW

**Comment by tgxworld** [comment]
> Whenever I select the benchmark type and release type the url gets created but it doesn’t showcase any output. How on refreshing the page we will get the required output. How do I solve this?

My suggestion is to do without javascript for now since you're not familiar with it. Start with just a form first.

**Comment by prathamesh-sonpatki** [comment]
I agree with Alan. Please work with simple form and then we will think about integrating Javascript.

**Comment by shahsaurabh0605** [comment]
I have made a simple form for `releases` which uses already built-in classes and css. Have a look at it.

File: app/controllers/admin_controller.rb [hunk 1]
```diff
@@ -4,8 +4,43 @@ class AdminController < ApplicationController
...
+   def next
+     if params[:result_type] && params[:admin][:releases_type]
+       @output = `sudo docker run --rm \
+         -e "RUBY_BENCHMARKS=true" \
+         -e "RUBY_MEMORY_BENCHMARKS=true" \
+         -e "RUBY_VERSION= #{params[:admin][:releases_type]}" \
+         -e "API_NAME=#{Rails.application.secrets.api_name}" \
+         -e "API_PASSWORD=#{Rails.application.secrets.api_password}" \
+         -e "INCLUDE_PATTERNS= #{params[:result_type]}"  rubybench/ruby_releases`
+     end
```
Comment (shahsaurabh0605) [inline app/controllers/admin_controller.rb]: What should I output when I submit the form and go to the next page?

**Pull request closed without merging**